pkg/volume/volume.go

```diff
@@ -4,49 +4,31 @@
 package volume
 
 import (
-	"bytes"
-	"context"
+	"regexp"
 
-	"github.com/Azure/acr-builder/pkg/procmanager"
+	"github.com/pkg/errors"
 )
 
-const (
-	// VolumePrefix is a prefix for volumes.
-	VolumePrefix = "acb_vol_"
-)
-
-// Volume describes a Docker volume.
+// Volume describes a Docker bind mounted volume.
 type Volume struct {
-	Name        string
-	procManager *procmanager.ProcManager
+	Name   string `yaml:"name"`
+	Source Source `yaml:",inline"`
 }
 
-// NewVolume creates a new Volume.
-func NewVolume(name string, pm *procmanager.ProcManager) *Volume {
-	return &Volume{
-		Name:        name,
-		procManager: pm,
+//Validate checks whether Volume is well formed
```

Comment:
nit: some of the code have a space after `//` and some of them doesn't...
can you add a space after `//` everywhere throughout the pr